validator/src/main/resources/expected-data-template/java/eks/aws-sdk-call-log.mustache

```diff
@@ -29,8 +29,8 @@
   "Version": "^1$",
   "RemoteService": "AWS::S3",
   "RemoteOperation": "GetBucketLocation",
-  "RemoteResourceIdentifier": "^e2e-test-bucket-name-{{testingId}}$",
-  "RemoteResourceType": "^AWS::S3::Bucket$",
+  "RemoteResourceIdentifier": "e2e-test-bucket-name-{{testingId}}",
+  "RemoteResourceType": "AWS::S3::Bucket",
   "Telemetry.Source": "^ClientSpan$",
```

Comment:
I just realized that we don't need to update these templates because this regex pattern is not for searching logs using logs filter, but actually look up the log entires after getting them from logs.

The change in validator should be good enough, but it's ok to do exactly match here. I don't mind.

Reply:
I think you would need to change the syntax if you wanted to use regex instead of exact match. I think the current implementation is fine.